src/App.tsx

```diff
@@ -55,6 +55,8 @@ export default function App() {
 		}
 	}, [markerFile]);
 
+	const [frameEnd, setEnd] = useState(markerFileData.frames.length);
```

Comment:
this doesn't work since the file hasn't been loaded when this initializes

I would have the initial value be 0 here and update it in an effect based on `markerFileData.frames.length`

Reply:
resolved

Reply:
well now it's just zero